dev-packages/localization-manager/package.json

```diff
@@ -29,7 +29,10 @@
     "watch": "theiaext watch"
   },
   "dependencies": {
+    "@types/bent": "^7.0.1",
     "@types/fs-extra": "^4.0.2",
+    "bent": "^7.1.0",
+    "colors": "^1.4.0",
```

Comment:
@msujew can we replace colors with `chalk` after https://github.com/eclipse-theia/theia/pull/10612?

Reply:
Rebased and replaced `colors` with `chalk` 👍